extensions/ql-vscode/src/variant-analysis/data-flow-paths-view.ts

```diff
@@ -0,0 +1,62 @@
+import { ExtensionContext, ViewColumn } from "vscode";
+import { AbstractWebview, WebviewPanelConfig } from "../abstract-webview";
+import { assertNever } from "../pure/helpers-pure";
+import { telemetryListener } from "../telemetry";
+import {
+  FromDataFlowPathsMessage,
+  ToDataFlowPathsMessage,
+} from "../pure/interface-types";
+import { DataFlowPaths } from "./shared/data-flow-paths";
+
+export class DataFlowPathsView extends AbstractWebview<
+  ToDataFlowPathsMessage,
+  FromDataFlowPathsMessage
+> {
+  public static readonly viewType = "codeQL.dataFlowPaths";
+
+  public constructor(ctx: ExtensionContext) {
+    super(ctx);
+  }
+
+  public async showDataFlows(dataFlowPaths: DataFlowPaths) {
+    if (!this.isShowingPanel) {
+      await this.getPanel();
+      await this.waitForPanelLoaded();
+    }
+
+    // Focus on panel
+    this.panel?.reveal(undefined, true);
+
+    await this.postMessage({
+      t: "setDataFlowPaths",
+      dataFlowPaths,
+    });
+  }
+
+  protected async getPanelConfig(): Promise<WebviewPanelConfig> {
+    return {
+      viewId: DataFlowPathsView.viewType,
+      title: "Data Flow Paths",
```

Comment:
Will you be updating this in a future PR to include the name of the query and potentially the name of the database/repository?

Reply:
I don't believe so - we won't be including that information AFAIK. That could potentially change once Product/Design reviews though.